include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -41,20 +41,28 @@ struct Exception : std::exception {
 // it transparent to the users of the API.
 template <typename T>
 struct Global {
-  static const OrtApi& api_;
+  static const OrtApi* api_;
 };
 
 #ifdef EXCLUDE_REFERENCE_TO_ORT_DLL
 OrtApi stub_api;
 template <typename T>
-const OrtApi& Global<T>::api_ = stub_api;
+const OrtApi* Global<T>::api_ = &stub_api;
```

Comment:
I think since you changed this to a pointer, you could just default initialize this (using {}) and then you can remove the stub_api stuff (as it's all zero anyways, but it was a simpler change at the time vs changing all the code to use a pointer).

So with that change you can remove EXCLUDE_REFERENCE_TO_ORT_DLL and just have ORT_API_MANUAL_INIT.

Reply:
is anyone already using `EXCLUDE_REFERENCE_TO_ORT_DLL` ? we don't want to break existing users right?